Dockerfile

```diff
@@ -2,17 +2,19 @@ FROM crystallang/crystal:0.36.1-alpine AS base
 
 WORKDIR /app
 
+RUN apk add --no-cache yaml-static
+
 COPY shard.yml .
 COPY shard.override.yml .
 COPY shard.lock .
 
-RUN shards install --static
+RUN shards install --production
```

Comment:
Definitely need the `--static` as there are compilation targets of some of the shards that need to be statically compiled.

Reply:
I don't think that flag has any effect on the `install` command: https://github.com/crystal-lang/shards/blob/addc26a3f22fee9fccb01cbb3e8878bef02d4295/src/cli.cr#L61-L64

Reply:
Previously resolved this via explicitly specifying build target of shards. Would be nice if this wasn't required.

Reply:
Build targets in shards don't compile as part of an install action (unless there's a post-install script which builds them, in which case that needs to specify the `--static` flag if that's required).

Worth noting that flag _is_ passed to the [build action here](https://github.com/PlaceOS/init/blob/2a20a1f3772025a042283b901e6d2ce9a1b39ed1/Dockerfile#L17) though.